xclim/indicators/atmos/_conversion.py

```diff
@@ -19,6 +19,10 @@ class Converter(Indicator):
     def validate(self, da):
         """Input validation."""
 
+    def missing(self, *args, **kwds):
+        """Return whether an output is considered missing or not."""
+        return False
```

Comment:
Not sure why we would skip checks on "_conversion" indicators? Usually we can trust xarray to do it anyway, but what if?  An invalid "tasmin" should imply an invalid "tas", and so on.

Reply:
Totally agree, but `missing` is intended for data reductions along time. In this case, we have a scalar operation (n -> n), so I assume there is no need for the missing function. This is a check I could add.

Reply:
Euh, but isn't that what this PR adds? Missing methods for non-reducing indicators?

Reply:
No, it's missing methods for indicators reducing on entire series (no resampling): n -> 1